src/gui/activitywidget.h

```diff
@@ -160,8 +91,21 @@ public slots:
     QPushButton *_copyBtn;
 
     QSet<QString> _accountsWithoutActivities;
+    QMap<Activity::Identifier, NotificationWidget*> _widgetForNotifId;
+    QElapsedTimer _guiLogTimer;
+    QSet<int> _guiLoggedNotifications;
+    ActivityList _blacklistedActivities;
```

Comment:
Maybe this needs to be a QSet otherwise your code might get very slow if there are many blacklisted activities (O(n²))